oc-includes/osclass/emails.php

```diff
@@ -706,18 +732,24 @@ function fn_email_new_comment_admin($aItem) {
             osc_item_url(),
             $itemURL
         );
+
         $title_email = osc_apply_filter('email_new_comment_admin_title_after', osc_mailBeauty(osc_apply_filter('email_title', osc_apply_filter('email_new_comment_admin_title', $content['s_title'], $aItem)), $words), $aItem);
         $body_email = osc_apply_filter('email_new_comment_admin_description_after', osc_mailBeauty(osc_apply_filter('email_description', osc_apply_filter('email_new_comment_admin_description', $content['s_text'], $aItem)), $words), $aItem);
 
-        $emailParams = array(
-            'from'      => osc_contact_email(),
-            'to'        => $admin_email,
-            'to_name'   => __('Admin mail system'),
-            'subject'   => $title_email,
-            'body'      => $body_email,
-            'alt_body'  => $body_email
-        );
-        osc_sendMail($emailParams);
+        $admins = Admin::newInstance()->listAll();
+        foreach($admins as $admin) {
```

Comment:
Hi Conejo, finally got at my PC and had time to look into this and expalin to you - why.
The reason is simple, in all other functions we send mails to users, not admins. But, in this case, we send them to admins only, and the "to" field would be empty (or the first admin from the instance would get 2 emails, unless I skip the array).

I know it seems like redundancy in the osc_send_mail() functions, but the code is very clear to read. If you think that it is better to use cc/bcc (can "to" field be empty in this case?), you can do the modification.